### PR TITLE
FIX - Keep form/modal when clicking outside the form

### PR DESCRIPTION
**Pull request by utchoang**
Open for https://github.com/apache/cloudstack-primate/issues/629
@rhtyd cc @svenvogel I've completed fix this task. Thanks.

### CODE REVIEW

**Comment by davidjumani** [comment]
@blueorangutan package

File: src/views/project/iam/ProjectRoleTab.vue [hunk 1]
```diff
@@ -103,7 +108,12 @@
             </span>
           </a-form>
         </a-modal>
-        <a-modal title="Create Project Role" v-model="createModalVisible" :footer="null" :afterClose="closeAction">
+        <a-modal
+          title="Create Project Role"
```
Comment (rhtyd) [inline src/views/project/iam/ProjectRoleTab.vue:112]: @Pearl1594 I think these are your feature - can you send another PR that fixes the translation keys?

File: src/views/project/iam/ProjectRoleTab.vue [hunk 2]
```diff
@@ -61,7 +61,12 @@
             </a-tooltip>
           </span>
         </a-table>
-        <a-modal title="Edit Project Role" v-model="editModalVisible" :footer="null" :afterClose="closeAction">
+        <a-modal
+          title="Edit Project Role"
```
Comment (rhtyd) [inline src/views/project/iam/ProjectRoleTab.vue:65]: @Pearl1594 I think these are your feature - can you send another PR that fixes the translation keys?

**Comment by rhtyd** [comment]
LGTM

**Comment by rhtyd** [comment]
@utchoang tip - when you send a PR that fixes an issue can you use `Fixes #id-of-pr` in the git commit or PR summary that way merging the PR will close the bug.
